flake.nix

```
@@ -358,6 +367,7 @@
                       order = super.auth.unix.order - 10;
                       control = "sufficient";
                       modulePath = "${cfg.package}/lib/libpam_himmelblau.so";
+                      settings.mfa_poll_prompt = cfg.mfaSshWorkaroundFlag;
```

One small suggestion 
```suggestion
                      settings.mfa_poll_prompt = cfg.mfaSshWorkaroundFlag && service == "sshd";
```